build.sbt

```diff
@@ -178,7 +178,8 @@ def mimaSettingsSince(versions: Seq[String]): Seq[Def.Setting[_]] = Def settings
     exclude[DirectMissingMethodProblem]("sbt.PluginData.this"),
     exclude[IncompatibleResultTypeProblem]("sbt.EvaluateTask.executeProgress"),
     exclude[DirectMissingMethodProblem]("sbt.Keys.currentTaskProgress"),
-    exclude[IncompatibleResultTypeProblem]("sbt.PluginData.copy$default$10")
+    exclude[IncompatibleResultTypeProblem]("sbt.PluginData.copy$default$10"),
+    exclude[DirectMissingMethodProblem]("sbt.Defaults.configArtifactPathSetting"),
```

Comment:
I have no idea why the exclusion was even needed. as`configArtifactPathSetting` is a `private[sbt]` method. But if I don't add it, MIMA complains.